src/tokens/base.json

```diff
@@ -262,5 +262,21 @@
     "symbol": "CLANKER",
     "decimals": 18,
     "logoURI": "https://coin-images.coingecko.com/coins/images/51440/large/CLANKER.png?1731232869"
+  },
+  {
+    "chainId": 8453,
+    "address": "0xcbD06E5A2B0C65597161de254AA074E489dEb510",
+    "name": "Coinbase Wrapped DOGE",
+    "symbol": "CBDOGE",
+    "decimals": 18,
```

Comment:
this one seeing 8 here:

<img width="495" alt="image" src="https://github.com/user-attachments/assets/48b6c82c-b5ba-492e-affc-af720388931c" />

https://basescan.org/token/0xcbD06E5A2B0C65597161de254AA074E489dEb510